test/core/python/pythonFromString.dyn

```diff
@@ -11,7 +11,6 @@
   "Nodes": [
     {
       "ConcreteType": "PythonNodeModels.PythonStringNode, PythonNodeModels",
-      "Engine": "CPython3",
```

Comment:
is this a modified .dyn or do we no longer serialize the engine for some reason?

Reply:
I removed the serialized engine so that the corresponding test can cover the case of the Default serialized value (more coverage in the test)

Reply:
if you modified it by hand maybe rename the file so that anyone in the future resaving this will aware something is hand modified.

Reply:
can do